app-office/texmacs/texmacs-2.1-r100.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+GUILE_COMPAT=( 1-8 )
+GUILE_REQ_USE="debug?,deprecated"
+inherit cmake xdg-utils guile-single
```

Comment:
You could do xdg instead and drop pkg_postinst/preinst.

Reply:
Pushed a fixup for this, will squash it once it's ready to merge.